### PR TITLE
Fixed #27373 -- Corrected message on 404 debug page for an empty request path.

### PR DESCRIPTION
**Pull request by felixxm**
https://code.djangoproject.com/ticket/27373

### CODE REVIEW

File: django/views/debug.py [hunk 1]
```diff
@@ -1193,7 +1193,10 @@ def default_urlconf(request):
           </li>
         {% endfor %}
       </ol>
-      <p>The current URL, <code>{{ request_path|escape }}</code>, didn't match any of these.</p>
+      <p>
+        The current URL {% if request_path %}, <code>{{ request_path|escape }}</code>{% endif %}, didn't match
```
Comment (timgraham) [inline django/views/debug.py]: It would be more correct without the comma in the empty case, and I think a test wouldn't hurt.

**Comment by raiderrobert** [comment]
Link to the ticket: https://code.djangoproject.com/ticket/27373

File: django/views/debug.py [hunk 2]
```diff
@@ -1193,7 +1193,11 @@ def default_urlconf(request):
           </li>
         {% endfor %}
       </ol>
-      <p>The current URL, <code>{{ request_path|escape }}</code>, didn't match any of these.</p>
+      <p>
+        {% if request_path %}
+        The current URL, <code>{{ request_path|escape }}</code>,{% else %}
```
Comment (timgraham) [inline django/views/debug.py]: URL->path would be more correct, I guess.

File: tests/view_tests/tests/test_debug.py [hunk 3]
```diff
@@ -115,6 +121,12 @@ def test_404_not_in_urls(self):
         self.assertNotContains(response, "Raised by:", status_code=404)
         self.assertContains(response, "<code>not-in-urls</code>, didn't match", status_code=404)
 
+    @override_settings(ROOT_URLCONF=WithoutEmptyPathUrls)
+    def test_404_empty_path_not_in_urls(self):
+        response = self.client.get('/')
+        self.assertNotContains(response, "Raised by:", status_code=404)
```
Comment (timgraham) [inline tests/view_tests/tests/test_debug.py]: Not sure if this line is important for this test?

**Pull request merged**